foam_controlwrapper/tests/test_controlwrapper.py

```diff
@@ -20,52 +18,117 @@
 class FoamControlWrapperTestCase(unittest.TestCase):
     """Test case for FoamControlWrapper class"""
     def setUp(self):
-        pass
 
-    def test_pitzDailyRun(self):
-        """Test to run pitzDaily laminar example in OpenFoam"
+        self.mesh = Mesh(name="mesh1")
+
+        self.points = [
+            Point(
+                (0.0, 0.0, 0.0),
+                data=DataContainer({CUBA.VELOCITY: [0, 0, 0]})),
+            Point(
+                (1.0, 0.0, 0.0),
+                data=DataContainer({CUBA.VELOCITY: [0, 0, 0]})),
+            Point(
+                (1.0, 1.0, 0.0),
+                data=DataContainer({CUBA.VELOCITY: [0, 0, 0]})),
+            Point(
+                (0.0, 1.0, 0.0),
+                data=DataContainer({CUBA.VELOCITY: [0, 0, 0]})),
+            Point(
+                (0.0, 0.0, 1.0),
+                data=DataContainer({CUBA.VELOCITY: [0, 0, 0]})),
+            Point(
+                (1.0, 0.0, 1.0),
+                data=DataContainer({CUBA.VELOCITY: [0, 0, 0]})),
+            Point(
+                (1.0, 1.0, 1.0),
+                data=DataContainer({CUBA.VELOCITY: [0, 0, 0]})),
+            Point(
+                (0.0, 1.0, 1.0),
+                data=DataContainer({CUBA.VELOCITY: [0, 0, 0]}))
+        ]
+
+        puids = [self.mesh.add_point(point) for point in self.points]
+
+        self.faces = [
+            Face([puids[0], puids[3], puids[7], puids[4]],
+                 data=DataContainer({CUBA.LABEL: 0})),
+            Face([puids[1], puids[2], puids[6], puids[5]],
+                 data=DataContainer({CUBA.LABEL: 1})),
+            Face([puids[0], puids[1], puids[5], puids[4]],
+                 data=DataContainer({CUBA.LABEL: 2})),
+            Face([puids[3], puids[2], puids[6], puids[7]],
+                 data=DataContainer({CUBA.LABEL: 3})),
+            Face([puids[0], puids[1], puids[2], puids[3]],
+                 data=DataContainer({CUBA.LABEL: 4})),
+            Face([puids[4], puids[5], puids[6], puids[7]],
+                 data=DataContainer({CUBA.LABEL: 5}))
+
+        ]
+
+        [self.mesh.add_face(face) for face in self.faces]
+
+        self.cells = [
+            Cell(puids)
+        ]
+
+        [self.mesh.add_cell(cell) for cell in self.cells]
+
+    def test_add_mesh(self):
+        """Test add_mesh method
+
+        """
+
+        wrapper = FoamControlWrapper()
+        wrapper.add_mesh(self.mesh)
+
+    def test_get_mesh(self):
+        """Test get_mesh method
+
+        """
+
+        wrapper = FoamControlWrapper()
+        wrapper.add_mesh(self.mesh)
+        mesh_inside_wrapper = wrapper.get_mesh(self.mesh.name)
+        self.assertEquals(self.mesh.name, mesh_inside_wrapper.name)
```

Comment:
Regarding FoamMesh, I see that edge-related methods and the ``add_xxx`` methods aren't supported at the moment.  For those methods of ABCMesh that are currently implemented, it would be good to test them here or better yet in the file `test_foam_mesh.py`